fleetspeak/src/server/components/components.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/google/fleetspeak/fleetspeak/src/server/admin"
 	"github.com/google/fleetspeak/fleetspeak/src/server/authorizer"
 	"github.com/google/fleetspeak/fleetspeak/src/server/comms"
-	log "github.com/golang/glog"
+	"github.com/golang/glog"
```

Comment:
Please keep the `log` alias for this import, it helps when synchronizing with googles internal project

Reply:
Reverted